invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -73,3 +69,37 @@ body {
 .padded-small {
   padding: 1rem 0 !important;
 }
+
+// Export page
+pre.export.result {
+    display:block;
+    white-space: pre-wrap;
+    padding:9.5px;
+    margin:0 0 10px;
+    font-size:13px;
+    line-height:1.42857;
+    word-break:break-all;
+    word-wrap:break-word;
+    background-color:#f5f5f5;
+    border:1px solid #ccc;
+    border-radius:4px;
+}
+
+.rdm-logo{
+  width: 250px;
+}
+
+.borderless{
+  border: 0 !important;
+}
+//.record-elem .record-title {
+//  color: #0377cd;
+//}
+//
+//.record-elem .record-description {
+//  color: #000000;
+//}
+//
+//.panel-free-title {
+//  color: #0377cd;
+//}
```

Comment:
Why adding commented css? Doesn't also seem to have a related class.

Reply:
because it is still work in progress and I will remove it later if I find no related element